website/common/locales/en/limited.json

```diff
@@ -55,11 +50,6 @@
     "seasonalItems": "Seasonal Items",
     "nyeCardAchievementTitle": "Auld Acquaintance",
     "nyeCardAchievementText": "Happy New Year! Sent or received <%= count %> New Year's cards.",
-    "nye0": "Happy New Year! May you slay many a bad Habit.",
-    "nye1": "Happy New Year! May you reap many Rewards.",
-    "nye2": "Happy New Year! May you earn many a Perfect Day.",
-    "nye3": "Happy New Year! May your To-Do list stay short and sweet.",
-    "nye4": "Happy New Year! May you not get attacked by a raging Hippogriff.",
```

Comment:
Ah ok, Thanks for the help :)

I've re added the following strings

-    "dateEndJuly": "July 29",
-    "dateEndAugust": "August 31",
-    "dateEndOctober": "October 31",
-    "dateEndNovember": "November 30",
-    "nye1": "Happy New Year! May you reap many Rewards.",
-    "nye2": "Happy New Year! May you earn many a Perfect Day.",
-    "nye3": "Happy New Year! May your To-Do list stay short and sweet.",
-    "nye4": "Happy New Year! May you not get attacked by a raging Hippogriff.",

and found that the pet strings you mentioned were part of a list called SpecialPets in the scripts file which is referenced in a few places. I've added them below and also added the strings back into the project.

\habitica\migrations\users\full-stable.js(75):  each(keys(content.specialPets), (pet) => {
\habitica\test\content\stable.test.js(50):  describe('specialPets', () => {
\habitica\test\content\stable.test.js(52):      each(stable.specialPets, (pet) => {
\habitica\test\content\stable.test.js(108):      let specialNumber = Object.keys(stable.specialPets).length;
\habitica\website\client\components\modifyInventory.vue(89):                li.list-group-item(v-for="(pet, value) in content.specialPets" ng-init="inv.pets[pet] = user.items.pets[pet]")
\habitica\website\client\components\inventory\stable\index.vue(116):        .btn.btn-flat.btn-show-more(@click="setShowMore(petGroup.key)", v-if='petGroup.key !== "specialPets"')
\habitica\website\client\components\inventory\stable\index.vue(612):            key: 'specialPets',
\habitica\website\client\components\inventory\stable\index.vue(614):              special: this.content.specialPets,
\habitica\website\client\components\inventory\stable\index.vue(707):          case 'specialPets':
\habitica\website\client\components\inventory\stable\index.vue(751):        let withProgress = isPetList && animalGroup.key !== 'specialPets';
\habitica\website\client\components\inventory\stable\index.vue(816):        if (animalGroup.key === 'specialPets') {
\habitica\website\common\script\content\index.js(289):api.specialPets = stable.specialPets;
\habitica\website\common\script\content\stable.js(54):let specialPets = {
\habitica\website\common\script\content\stable.js(94):each(specialPets, (translationString, key) => {
\habitica\website\common\script\content\stable.js(117):  specialPets,
\habitica\website\server\models\user\schema.js(317):      _.transform(shared.content.specialPets, (m, v, k) => m[k] = Number),

I've also pushed the changes, would you like me to create a new pull request?

Reply:
@Inshabel: You could simply [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) this branch and fix up the commits, like for example if your current branch is the `develop` branch it's just a `git rebase -i master`. It should open an editor where you can mark to alter the commits in several ways (listed in the file that is displayed in the editor) and one of them is `fixup` (marking it like this can also be done when creating the commit: `git commit --fixup <other-commit-id>`), which merges the changes in the marked commits with its parent. Rebasing against the current master should also give you the chance to [fix](https://developer.atlassian.com/blog/2015/12/tips-tools-to-solve-git-conflicts/) the conflicts.

Reply:
Ah thanks @aszlig . Do I still need to do that if I've just made the changes already and pushed to my repo?

Reply:
Btw. you can also join the [Aspiring Blacksmiths (Habitica Coders)](https://habitica.com/groups/guild/68d4a01e-db97-4786-8ee3-05d612c5af6f) guild (if you didn't do so already) if you have further questions about git. Further documentation can be also found at https://git-scm.com/book/en/v2.